docusign-event-gw/internal/outgoing/outgoing.go

```diff
@@ -66,3 +77,25 @@ func (e *EventForwarder) Forward(event *events.KymaEvent) (map[string]interface{
 	return respMap, nil
 
 }
+
+func (e *EventForwarder) enrichRequest(event *events.CloudEvent, request *http.Request) *http.Request {
+
+	eventId, _ := generateEventID()
+	//sourceID := (config.GlobalConfig.AppName).String()
+
+	request.Header.Set("Content-Type", "application/cloudevents+json; charset=utf-8")
+	request.Header.Set("ce-specversion", "1.0")
+	request.Header.Set("ce-type", event.EventType)
+	request.Header.Set("ce-eventtypeversion", "v1")
+	request.Header.Set("ce-id", eventId)
+	request.Header.Set("ce-source", "docusign")
```

Comment:
is this intentional to set `ce-source` permanently as `docusign` and not the application name?

Reply:
changed back to take it from config